backend/apps/matching/views/staff_approvals.py

```diff
@@ -21,6 +22,13 @@ class ApproveParticipantsView(View):
     template_name = "approve_hospitals.html"
 
     def get(self, request, p_type):
+        # If you are not a staff user, the method decorator takes care of showing a 404 page,
```

Comment:
I don't get a 404 as fake user A1:
![image](https://user-images.githubusercontent.com/3034832/87227946-55cd7a00-c39e-11ea-855b-2e8c0edb6a8a.png)

Reply:
Good catch - the django decorator `staff_member_required` redirects to the login page under the assumption that you need to log in again with the higher ranked user.

You can override this behaviour like I did in cc5057d and 892abf0, but one might forget this. Under the assumption that we want to show as little information as possible about our internal url structures (debatable as an open source plattform), I added a customized version of the `staff_member_required` decorator, which has this behavior built-in, but also customizable. Let me know what you think!

Reply:
Is there a possibility to override `staff_member_required` with our version project-wide?

Reply:
We could give it the same name and import it from our repo, but I thought that might be too easy to miss in a future PR (to check whether the import is right when the decorator is used later in the file). I don't think we can override an import from a django package.

Reply:
We could in theory add a test for it though - but that might be overkill.